examples/production_monitoring/README.md

```diff
@@ -0,0 +1,81 @@
+# vLLM + Prometheus/Grafana 
+
+This is a simple example that shows you how to connect vLLM metric logging to the Prometheus/Grafana stack.
+
+> For this example, we launch Prometheus and Grafana via Docker. Make sure you have [`docker`](https://docs.docker.com/engine/install/) and [`docker compose`](https://docs.docker.com/compose/install/linux/#install-using-the-repository) installed.
+
+### Launch
+
+Prometheus metric logging is enabled by default in the OpenAI-compatible server. Launch vLLM as usual:
+```bash
+python3 ../../vllm/entrypoints/openai/api_server.py --model mistralai/Mistral-7B-v0.1 --max-model-len 2048
+```
+
+Launch Prometheus and Grafana servers with `docker compose`:
+```bash
+docker compose up
+```
+
+Submit some sample requests to the server to generate some data:
+```bash
+python3 simulate_clients.py
```

Comment:
We would encourage users to use `benchmark_serving.py`. The command will be 

```
python benchmark_serving.py --dataset ./ShareGPT_V3_unfiltered_cleaned_split.json --model meta-llama/Llama-2-7b-chat-hf --num-prompts 20 --endpoint /v1/completions --tokenizer meta-llama/Llama-2-7b-chat-hf
```

After #2172 is merged. I might also add a way to randomly generate the input so it's no longer relied on datasets. In anyway, before we merge this PR, we should no longer need to include simulate_clients.py.

Reply:
sounds good 

note @simon-mo -- I'm working on simplifying this per comments above. I have a local branch I will be pushing up this weekend